core/templates/domain/utilities/language-util.service.ts

```diff
@@ -87,7 +87,8 @@ export class LanguageUtilService {
   }
 
   getSupportedContentLanguages(): {[languageCode: string]: ContentLanguage} {
-    const supportedContentLanguages = {};
+    const supportedContentLanguages:
+    {[languageCode: string]: ContentLanguage} = {};
```

Comment:
```suggestion
    const supportedContentLanguages:
      { [languageCode: string]: ContentLanguage } = {};
```

Reply:
Also probably extract the type into a separate variable.

Reply:
done